### ISSUE TITLE
KIO error when clicking on profile

### ISSUE THREAD

**Issue by kjcole** [ssbc/patchwork#1203]
I'm running a current Arch Linux system with

* ssb-patchwork 3.16.2-1
* kio 5.64.0-1
* kio-extras 19.08.3-1

When I clicked on my profile, I got a pop-up:

```
ERROR: KIO Client
Unable to run the command specified. The file or folder /opt/ssb-patchwork/assets/@... does not exist.
```
(where "..." is a hash.)

It did then show my profile, but I have no idea what the system wanted to do in addition to showing the profile...

**Comment by christianbundy** [comment]
Thanks for the bug report.

Does this happen consistently? If so, could you check whether this happens when you run Patchwork from source?

> (where "..." is a hash.)

Does this end in `.sha256` or `.ed25519`? I'd be interested to know what comes up when you search the full hash in Patchwork's search bar.

**Comment by kjcole** [comment]
The error is consistent and actually shows up when I click [Profile] and then click on my username in the small pop-up that appears below the [Profile] button.

The hash ends with  `ed25519`.

**Comment by christianbundy** [comment]
Any chance the hash is the same as your profile hash? I don't know why it would be looking for a file with the same name as a profile ID, but that would make more sense than looking for someone else's profile ID as a file.

When you go to other people's profiles can you reproduce the error? If so, does it output *their* ID?

**Comment by kjcole** [comment]
> Any chance the hash is the same as your profile hash?

It is indeed. (And actually, it's when I hover / mouse-over the [Profile] button that the wee pop-up appears. Not when I actually click the button.

> When you go to other people's profiles can you reproduce the error? If so, does it output _their_ ID?

Reproducible with other IDs. Hovering over your name in the "send private message" right-hand sidebar, and then clicking on your name in the wee pop-up produced:

    Unable to run the command specified. The file or folder /opt/ssb-patchwork/assets/@+oaWWDs8g73EZFUMfW37R/ULtFEjwKN/DczvdYihjbU=.ed25519 does not exist.

**Comment by christianbundy** [comment]
Hmm, I bet Electron is trying to pre-fetch the page or something silly.

I've put together a fix in #1204 but would love help testing it. Since I can't reproduce the error on my machine I'm limited to a guess-and-check bugfix.
